# coding: utf-8

from the_tale.game.map.exceptions import MapException

class PlacesException(MapException): pass
